app/controllers/charges_controller.rb

```diff
@@ -72,14 +69,16 @@ def charge_params
       :seller_id,
       :idempotency_key,
       :is_subscribed,
+      :campaign_id,
+      # TODO(justintmckibben): Deprecate this boolean in favor of campaign_id
```

Comment:
FYI @oliviasztanga @danthaman44 for GAM purchases, you have to pass the campaign_id instead of the boolean is_distribution on the frontend. is_distribution will still work, but it will just find the first Campaign that it finds